client/components/pie-chart/data/index.js

```diff
@@ -0,0 +1,27 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { isEqual, sortBy } from 'lodash';
+
+const NUM_COLOR_SECTIONS = 3;
+
+const assignSectionFunc = ( datum, index ) => {
+	return {
+		...datum,
+		sectionNum: index % NUM_COLOR_SECTIONS,
+	};
+};
+
+const sortDataAndAssignSections = data => {
+	return sortBy( data, datum => datum.value )
+		.reverse()
+		.map( assignSectionFunc );
+};
+
+const isDataEqual = ( currentData, newData ) => {
+	return isEqual( currentData, newData );
+};
+
+export { sortDataAndAssignSections, isDataEqual };
```

Comment:
I think we prefer to `export` right on the line of the definition these days, i.e.

```js
const sortDataAndAssignSections = data => {
	/* sortDataAndAssignSections implementation */
}
```

Reply:
👍 I will go in and change it when I get a chance in an adjacent PR.

Reply:
Much appreciated! Feel free to ping me for review!